gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/java/io/gravitee/rest/api/management/v2/rest/resource/group/GroupsResource.java

```diff
@@ -75,6 +84,25 @@ public GroupsResponse listGroups(@BeanParam @Valid PaginationParam paginationPar
             .links(computePaginationLinks(groups.size(), paginationParam));
     }
 
+    @POST
+    @Path("/_search")
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Produces(MediaType.APPLICATION_JSON)
+    @Permissions({ @Permission(value = RolePermission.ENVIRONMENT_GROUP, acls = { RolePermissionAction.READ }) })
+    public GroupsResponse searchGroupsByIds(@BeanParam PaginationParam paginationParam, @Valid GroupSearchParams searchParams) {
```

Comment:
```suggestion
    public GroupsResponse searchGroups(@BeanParam PaginationParam paginationParam, @Valid GroupSearchParams searchParams) {
```
This can be generic which can allow us to add different search parameters in the future (even though for now we are only searching by ids).

Reply:
I have updated the GroupSearchParams schema.